sycl/source/detail/queue_impl.cpp

```diff
@@ -746,6 +748,21 @@ void queue_impl::doUnenqueuedCommandCleanup(
     tryToCleanup(MDefaultGraphDeps);
 }
 
+void queue_impl::verifyProps(const property_list &Props) const {
+  static const auto AllowedPropList = GenerateAllowedProps<
+      sycl::ext::intel::property::queue::compute_index,
+      sycl::property::queue::in_order, sycl::property::queue::enable_profiling,
+      sycl::ext::oneapi::property::queue::discard_events,
+      sycl::ext::oneapi::property::queue::priority_normal,
+      sycl::ext::oneapi::property::queue::priority_low,
+      sycl::ext::oneapi::property::queue::priority_high,
+      sycl::ext::intel::property::queue::no_immediate_command_list,
+      sycl::ext::intel::property::queue::immediate_command_list,
+      sycl::ext::oneapi::cuda::property::queue::use_default_stream,
+      sycl::ext::codeplay::experimental::property::queue::enable_fusion>();
```

Comment:
Instead of having to maintain another list of properties, could we maybe use sycl/include/sycl/properties/queue_properties.def and the like to do this? I see that `enable_fusion` isn't in that .def file, but it could probably be moved there too.

Reply:
yeah, extra list of properties is my main concern about my implementation.

Reply:
regarding def files approach - should we keep basic properties and extensions split in terms of files or it is not a big deal? Could we mix them?